test/fail_compilation/fail19912b.d

```diff
@@ -0,0 +1,9 @@
+// PERMUTE_ARGS:
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19912b.d(8): Error: class `fail19912b.object` conflicts with import `fail19912b.object` at fail_compilation/fail19912b.d
```

Comment:
The error message mentions `import`, but there is `import` in the source code. Can we come up with a better message?